src/jit/codegenlegacy.cpp

```diff
@@ -5646,12 +5646,10 @@ void CodeGen::genCodeForQmark(GenTreePtr tree, regMaskTP destReg, regMaskTP best
             // If any candidates are not alive at the GT_QMARK node, then they
             // need to be spilled
 
-            VARSET_TP VARSET_INIT(compiler, rsLiveNow, compiler->compCurLife);
-            VARSET_TP VARSET_INIT_NOCOPY(rsLiveAfter, compiler->fgUpdateLiveSet(compiler->compCurLife,
-                                                                                compiler->compCurLifeTree, tree));
+            VARSET_VALARG_TP rsLiveNow(compiler->compCurLife);
```

Comment:
>VARSET_VALARG_TP [](start = 12, length = 16)

I think the VARSET_VALARG_TP is only intended to be used in a function signature, not to declare variables.